templates/patterns/search_block/pattern-search-block.html.twig

```diff
@@ -28,9 +28,13 @@
   {% endfor %}
 {% endif %}
 
+{% set attributes = attributes.setAttribute('method', 'get') %}
```

Comment:
since it was established that we dont theme form elements we dont use this anymore and should be deleted (the entire pattern)